com.syntevo.SmartSynchronize.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Categories=Development;
+Comment=File and Directory Compare
+Encoding=UTF-8
+Exec=smartsynchronize
+Icon=com.syntevo.SmartSynchronize
+Keywords=compare;merge
```

Comment:
Ends in `;` again

Reply:
Since this is a _copycat_ of `SmartGit` I'll address all the comments once the other one is done.